core/templates/dev/head/pages/topics_and_skills_dashboard/merge_skill_modal_directive.html

```diff
@@ -0,0 +1,25 @@
+<div class="modal-header">
+  <h3>Select the skill to merge into</h3>
+</div>
+
+<div class="modal-body">
+  <p>
+    Merging into skill: <[selectedSkill.description]>
+  </p>
+  <div class="skills">
+    <skills-list skill-summaries="skillSummaries" in-modal="true"
+                 selected-skill="selectedSkill">
```

Comment:
It might be better to pass selected-skill-id instead

Reply:
Ah, no, I was thinking that we already have the data in skillSummaries and can construct a key-value map (id --> description) from there, then just retrieve by key. Definitely agree that another call to the backend isn't a good idea.

That said, the tradeoff isn't that strong in this case (I get the point about wanting to keep the code simple, too), so I retract my original comment. Thanks!